ReactiveCocoa/CocoaAction.swift

```diff
@@ -37,9 +35,11 @@ public final class CocoaAction<Sender>: NSObject {
 			producer.start()
 		}
 
-		isEnabled = action.isEnabled
-		isExecuting = action.isExecuting
-		
+		isEnabled = Property(initial: action.isEnabled.value,
+		                     then: action.isEnabled.producer.observe(on: UIScheduler()))
```

Comment:
These can be replaced with https://github.com/ReactiveCocoa/ReactiveSwift/pull/157 once merged.